src/serve/index.ts

```diff
@@ -24,10 +24,10 @@ export async function serve(options: any): Promise<void> {
   options.port = parseInt(options.port, 10);
   if (targetNames.includes("hosting") && config.extract(options).some((it: any) => it.source)) {
     experiments.assertEnabled("webframeworks", "emulate a web framework");
-    
+
     // TODO: pass emulator info as the fourth parameter so that the dev server
     //       can use host and port
-    await prepareFrameworks(targetNames, options, options); 
+    await prepareFrameworks(targetNames, options, options);
```

Comment:
Could we pass something in the shape of an emulatorInfo here for now? e.g, `[{name: 'hosting', port: options.port, ... }]`

Reply:
I harcoded it and changed the TODO to get the "real emulator info"